Regression18.dfy(20,10): Error: value does not satisfy the subset constraints of 'NewT'
Execution trace:
    (0,0): anon0

Dafny program verifier finished with 5 verified, 1 error
